packages/steiger-plugin-fsd/src/no-cross-imports/README.md

```diff
@@ -0,0 +1,108 @@
+# `no-cross-imports`
+
+This rule forbids cross-imports between slices on the same layer. This is in accordance to the slice isolation principle in Feature-Sliced Design:
```

Comment:
issue: this might be a little confusing because this is not the terminology used in the documentation

suggestion: let's use "the import rule on layers" instead
```suggestion
This rule forbids cross-imports between slices on the same layer. This is in accordance to the import rule on layers:
```